### DISCUSSION TITLE
Schedule Jobs are froozen

### DISCUSSION THREAD

**Discussion by tinkerbellbasf85** [MISP/MISP#9669] [General]
Hello all,

I have some schedule task that are frozen in 75%. When the jobs are running manually are working fine, but when they are scheduled they don't finish correctly.
Could you mind to help me, please? or Where i can find the logs to investigate the issue? 
Thanks

**Comment by iglocska** [comment]
Generally the scheduler is hot garbage, best to use cron jobs for scheduled tasks rather than the built in one.

**Reply by tinkerbellbasf85** [reply]
which user is going to run the crontab? because I have checked the crontab job but there are not a lot of jobs.... Thanks for your quick replay

**Reply by iglocska** [reply]
I'd use the apache user for it (www-data or apache depending on your distro). Have a look at the automation page in MISP for a description of how to configure cron jobs for MISP.

**Reply by tinkerbellbasf85** [reply]
Thanks I will take a look
